packages/gatsby-theme/src/blocks/ContactBlock/ContactBlock.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import ContactBlock from "."
+
+export const contactInfo = [
+  {
+    title: "Email",
+    icon: "comment",
+    text: "hq@octahedroid.com",
+  },
+  {
+    title: "Schedule",
+    icon: "calendar",
+    text: `<a
+              href="https://calendly.com/jmolivas/free-one-to-one-consultation"
+              target="_blank"
+              rel="noreferrer"
+              }}
+            >
+              Free consultation
+            </a>`,
+  },
+  {
+    title: "Phone",
+    icon: "phone",
+    text: "760-618-1215",
+  },
+]
+
+export const contactArgTypes = {
+  align: {
+    description: "Text align",
+    control: {
+      type: "select",
+      options: ["left", "center", "right"],
+    },
+  },
+}
+
+<Meta title='blocks/ContactBlock' component={ContactBlock} />
+
+# ContactBlock
+
+You can use it to add a Contact block
+
+## Props
+
+<Props of={ContactBlock} />
+
+export const ContactBlockTemplate = (args) => <ContactBlock {...args} />
+
+### Default ContactBlock
+
+<Preview>
+  <Story
+    name='Default'
+    args={{
+      heading:
+        "Tell us about your project. And we will contact you promptly to discuss next steps.",
+      contactInfo,
+    }}
+    argTypes={contactArgTypes}
+  >
```

Comment:
The argTypes object should be passed to the Meta tag instead. This allows the argTypes object to be applied to all the stories. In this case it is a single story, but we should change this to standardize future implementations.
```suggestion
<Meta title='blocks/ContactBlock' component={ContactBlock} argTypes={contactArgTypes} />

# ContactBlock

You can use it to add a Contact block

## Props

<Props of={ContactBlock} />

export const ContactBlockTemplate = (args) => <ContactBlock {...args} />

### Default ContactBlock

<Preview>
  <Story
    name='Default'
    args={{
      heading:
        "Tell us about your project. And we will contact you promptly to discuss next steps.",
      contactInfo,
    }}
  >
```

Reply:
done @dinartecc